src/main/java/edu/cornell/kfs/tax/dataaccess/impl/TransactionRowDvBuilder.java

```diff
@@ -331,7 +334,38 @@ void buildTransactionRows(ResultSet rs, PreparedStatement insertStatement, T sum
         prepareForSecondPass(summary, docIds);
     }
 
+    /**
+     * Overridden to also insert nulls for the following field placeholders:
+     *
+     * <ul>
+     *   <li>disbursementNbr</li>
+     *   <li>paymentStatusCode</li>
+     * </ul>
+     *
+     * @see edu.cornell.kfs.tax.dataaccess.impl.TransactionRowBuilder#insertNullsForTransactionRow(java.sql.PreparedStatement,
+     * edu.cornell.kfs.tax.dataaccess.impl.TaxTableRow.TransactionDetailRow, int)
+     */
+    @Override
+    void insertNullsForTransactionRow(PreparedStatement insertStatement, TransactionDetailRow detailRow, int offset) throws SQLException {
+        super.insertNullsForTransactionRow(insertStatement, detailRow, offset);
+        insertStatement.setString(detailRow.disbursementNbr.index - offset, null);
+        insertStatement.setString(detailRow.paymentStatusCode.index - offset, null);

```

Comment:
The DV Builder doesn't appear to explicitly handle the new disbursementTypeCode field. If it's supposed to be null for DVs, then please add it to the insert-nulls method and update the method's Javadocs accordingly.

Reply:
Please see my latest commits